make/kube-dist

```diff
@@ -1,38 +1,47 @@
 #!/usr/bin/env bash
 
 set -o errexit -o nounset
+set -x
```

Comment:
Do we want this?

Reply:
Nope; removed in bc206b25d8f09322bffb700b7ba72da8783a823c